src/main/java/com/googleinterns/zoomtube/servlets/DeleteTranscriptServlet.java

```diff
@@ -0,0 +1,35 @@
+
+package com.googleinterns.zoomtube.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.googleinterns.zoomtube.data.TranscriptLine;
+import java.io.IOException;
+import java.util.stream.StreamSupport;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that deletes all of the transcript lines. */
+// TODO: Delete this file after testing is complete.
```

Comment:
I am not sure why you need to delete this after testing is complete?

Reply:
Users shouldn't have the ability to delete lecture transcripts from the datastore.

Reply:
Do we want to keep this code because Admins might want to delete transcripts?

Reply:
We currently don't have a feature planned for this. I think we could leave the code for deleting the transcript for now then.

Removed TODO.